src/transformers/training_args.py

```diff
@@ -944,8 +944,8 @@ class TrainingArguments:
             )
         },
     )
-    sharded_ddp: str = field(
-        default="",
+    sharded_ddp: Optional[Union[str, bool, List[ShardedDDPOption]]] = field(
+        default=False,
```

Comment:
The default shouldn't be touched.

Reply:
the documentation says that it defaults to False - sharded_ddp (`bool`, `str` or list of [`~trainer_utils.ShardedDDPOption`], *optional*, defaults to `False`). but if the default variable has to be " ", I'll try doing it some other way.

Reply:
Then let's fix the documentation as a change in default is potentially braking.

Reply:
Okay, changing default to `""` in the documentation as well as in the code.